plugins/devtools-backend/README.md

```diff
@@ -0,0 +1,60 @@
+# DevTools Backend
+
+Welcome to the DevTools backend plugin! This plugin provides data for the DevTools frontend features.
+
+## Setup
+
+The following sections will help you get the DevTools Backend plugin setup and running.
+
+### Up and Running
+
+Here's how to get the backend up and running:
```

Comment:
this subheading is redundant, it seems. there are no other level 3 headings here, so i'd just simplify it with the two sentences together and then steps!

Reply:
Yes, that seems better to me, I've adjusted this 👍